turbo.json

```diff
@@ -0,0 +1,8 @@
+{
+  "$schema": "https://turbo.build/schema.json",
+  "pipeline": {
+    "build": {
+      "outputs": [".next/**", "!.next/cache/**"]
+    }
+  }
+}
```

Comment:
Should we add `rome` commands to turbo.json as well?

Reply:
Yep i will take a look for this